src/pyclass/create_type_object.rs

```diff
@@ -22,7 +22,7 @@ where
 {
     unsafe {
         PyTypeBuilder::default()
-            .type_doc(T::DOC)
+            .type_doc(T::doc(py).unwrap()) // FIXME remove unwrap
```

Comment:
Why not just `?` as `create_type_object` by now returns `PyResult<..>`? Is an early return problematic inside the unsafe block?

Reply:
Ah yes, thanks, missed this was now fixable post-rebase!